deeplearning4j-scaleout/dl4j-streaming/src/main/spark-2/org/deeplearning4j/streaming/pipeline/spark/StreamingContextUtils.java

```diff
@@ -0,0 +1,27 @@
+package org.deeplearning4j.streaming.pipeline.spark;
+
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.function.VoidFunction;
+import org.apache.spark.streaming.api.java.JavaDStream;
+import org.apache.spark.streaming.api.java.JavaStreamingContext;
+
+/**
+ * In order to handle changes between Spark 1.x and 2.x
+ */
+public class StreamingContextUtils {
+
+    public static void awaitTermination(JavaStreamingContext jssc, long timeout) {
+        try {
+            if(timeout < 0)
+                jssc.awaitTermination();
+            else
+                jssc.awaitTerminationOrTimeout(timeout);
+        } catch (InterruptedException e) {
```

Comment:
Do we ?  `awaitTerminationOrTimeOut` has the same signature in Spark 1.6, & 2.x. AFAICT

https://spark.apache.org/docs/2.1.0/api/java/org/apache/spark/streaming/StreamingContext.html#awaitTerminationOrTimeout(long)
https://spark.apache.org/docs/1.6.2/api/java/org/apache/spark/streaming/StreamingContext.html#awaitTerminationOrTimeout(long)